applications/nrf_desktop/src/modules/hids.c

```diff
@@ -36,8 +36,13 @@ static size_t report_index[REPORT_ID_COUNT];
 BT_GATT_HIDS_DEF(hids_obj,
 		 REPORT_SIZE_MOUSE,
 		 REPORT_SIZE_KEYBOARD_KEYS,
-		 REPORT_SIZE_KEYBOARD_LEDS,
-		 REPORT_SIZE_CONSUMER_CTRL
+		 REPORT_SIZE_KEYBOARD_LEDS
+#if CONFIG_DESKTOP_HID_SYSTEM_CTRL
+		 , REPORT_SIZE_CTRL
+#endif
+#if CONFIG_DESKTOP_HID_CONSUMER_CTRL
```

Comment:
Should this be || ?

Or is it supported to have two _CTRL entires here?

Reply:
It can have 2 HID control reports (consumer control and system control) - that means two _CTRL entries here.